lib/ansible/config/base.yml

```diff
@@ -1790,4 +1790,18 @@ VERBOSE_TO_STDERR:
     - section: defaults
       key: verbose_to_stderr
   type: bool
+ERIC_ECCLI_ADDITIONAL_RE_FILE:
```

Comment:
This absolutely does not belong here. This should probably be moved to an option of the terminal plugin, though I am not sure why you need tunable regexes in the first place.

Reply:
It will output some prompt information when running command through ECCLI model. In order to capture these custom prompt and we will add matching rule in configuration file instead of writing  code in terminal plugin. I think it is more convenient and friendly for user of ansible, what do you think about?